Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node as react-build
+COPY . /code
+WORKDIR /code
+RUN cd reana-ui && \
+    yarn && \
+    yarn build
+
+FROM nginx
+COPY --from=react-build /code/reana-ui/build /usr/share/nginx/html
+EXPOSE 80
```

Comment:
Is this already exposed to the local host? I can access the ui web page only after `kubectl describe service` and friends... Thinking of nodeport, clusterip, rafeik and firiends whether they are ready? 

 But I can get there fine :+1: BTW the login page could say instead of "New to us? Ask for a token" something more neutral such as "New user? Request a token"